### PR TITLE
Move to browserify

### PR DESCRIPTION
**Pull request by tarrsalah**
This PR will add `boostrap` and `markdown-js` as npm dependencies, and use `browserify` with `gulp`  for better structure and clarity.

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -1,36 +1,32 @@
...
+var uglify = require('gulp-uglify');
+var browserify = require('browserify');
+var source = require('vinyl-source-stream');
+var buffer = require('vinyl-buffer');
+var concat = require('gulp-concat');
+var rename = require('gulp-rename');
+var minifyCSS = require('gulp-minify-css');
+var stripCssComments = require('gulp-strip-css-comments');
+
+gulp.task('css', function() {
```
Comment (ahmadajmi) [inline gulpfile.js]: let's keep the comments there :)

**Comment by ahmadajmi** [comment]
This is perfect, I really like your way of doing it, things are more simpler now :)

**Comment by tarrsalah** [comment]
@ahmadajmi  any other modifications to add before the merge?

**Comment by ahmadajmi** [comment]
@tarrsalah a small edit in the gulp.js file for comments

**Comment by ahmadajmi** [comment]
Great, one more thing is to squash the 3 commits into one so 

```
Move to browserify
add comments for gulp tasks.
Comments, again
```

become one commit, this will help decreasing commits and merging all changes in one clean and final commit 

this guide will help http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

**Comment by tarrsalah** [comment]
@ahmadajmi  done.

**Comment by ahmadajmi** [comment]
no need for theses two lines :)

```
add comments for gulp tasks.
Comments, again
```

`git commit --amend` will enable you to edit the last commit

**Comment by tarrsalah** [comment]
Oh yeah, forget about the commit message :+1:

**Comment by ahmadajmi** [comment]
Perfect 👍

**Comment by ahmadajmi** [comment]
Good job Salah and I 🚀 pushed it remotely :)

**Comment by tarrsalah** [comment]
Nice :)

**Pull request merged**